src/lib.rs

```diff
@@ -27,11 +176,9 @@ extern crate std;
 #[doc(hidden)]
 pub use core as __core;
 
-/// The `bitflags!` macro generates a `struct` that holds a set of C-style
-/// bitmask flags. It is useful for creating typesafe wrappers for C APIs.
+/// The macro used to generate the flag structure.
 ///
-/// The flags should only be defined for integer types, otherwise unexpected
-/// type errors may occur at compile time.
+/// See the [crate level docs](https://docs.rs/bitflags) for complete documentation.
```

Comment:
Whoops, I should comment here ^^

`../index.html` did not work for me locally, but `../bitflags/index.html` did. Is it fine if I add a commit for the latter?